src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/NewCaseReceiver.java

```diff
@@ -104,10 +109,17 @@ public void receiveNewCase(Message<byte[]> message) {
       }
     }
 
+    Map<String, String> sample = newCasePayload.getSample();
+
+    if (collex.getSurvey().getSampleDefinitionUrl().endsWith("business.json")) {
```

Comment:
Be nice to not do a string compare on a url

Reply:
I'd prefer something like a new type/setting on the survey?

Reply:
I think "inspired by" would be misleading, but this technique is being used in the INT team: https://github.com/ONSdigital/sdc-int-rh-service/blob/e8d8b09961e3273c522c2a9bfd4d58d01d5aff26/src/main/java/uk/gov/ons/ctp/integration/rhsvc/event/impl/CaseEventReceiverImpl.java#L55

"Survey type" is contentious, because it would require code changes every time there's a new "type" which we are going to great lengths to avoid.

The business integration remains tactical at this stage, so I think the check is adequate to keep the RAS-RM code separate from the rest.